wcc-contentful-app/app/helpers/wcc/contentful/app/section_helper.rb

```diff
@@ -31,6 +31,8 @@ def section_id(section)
   end
 
   def markdown(text)
+    return unless text
```

Comment:
I don't think I like this.  I'd rather it be fixed in the view.  Having the `markdown` helper blow up makes us actually take a look at the view and decide what we want to do - do we want the field to be required in contentful?  Do we want to have a fallback?  etc.